examples/analyse_counterpoise.py

```diff
@@ -4,51 +4,51 @@
 TODO: Needs high level description
```

Comment:
Can the description be added?

Reply:
Since changes to `test_analyse_counterpoise` made their way to this pull request, either remove the changes from this pull request and open a new one or resolve the documentation issues.

Reply:
Description added.